src/pages/trade-types/margin/index.js

```diff
@@ -6,13 +6,23 @@ import { SEO } from 'components/containers'
 import { Header } from 'components/elements'
 import Layout from 'components/layout/layout'
 import { localize, WithIntl } from 'components/localization'
+import TradingImage from 'images/common/og_deriv.png'
 const WhyTradeMargin = Loadable(() => import('./_why-trade-margin'))
 const Contracts = Loadable(() => import('./_contracts'))
 const Policies = Loadable(() => import('./_policies'))
 const StartTrading = Loadable(() => import('./_start-trading'))
 const RelationshipWithLeverge = Loadable(() => import('./_relationship-with-leverge'))
 const AvailableMarkets = Loadable(() => import('./_available-markets'))
 
+const meta_attributes = {
+    og_title: 'Margin trading | Trade types | Deriv',
+    og_description:
+        'Learn about margin trading on Deriv MT5 (DMT5). Enjoy 24/7 trading, high leverage, and zero commissions on forex, synthetic indices, and other assets.',
+    og_type: 'website',
+    og_img_width: '600',
+    og_img_height: '315',
+    og_img: TradingImage,
```

Comment:
You won't need to pass this in since it defaults to these values.

Reply:
Ok